### ISSUE TITLE
Vagrant up not working

### ISSUE THREAD

**Issue by strets123** [thesgc/chembiohub_helpdesk#748]
Need to fix our vagrant box then we can give the script to Chembl

We have someone waiting on this...

**Comment by strets123** [comment]
Now fixed as evidenced by comment here

https://github.com/thesgc/chembiohub_ws/issues/3

I have reopened a new issue as the user could not install on Windows and we need to fix this

**Comment by strets123** [comment]
see #768
